packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.controller.js

```diff
@@ -28,6 +29,14 @@ class LogsHomeCtrl {
     this.LogsHelperService = LogsHelperService;
     this.LogsDetailService = LogsDetailService;
     this.LogsConstants = LogsConstants;
+    this.IndexApiService = OvhApiDbaas.Logs()
+      .Index()
+      .v6();
+    this.AliasApiService = OvhApiDbaas.Logs()
+      .Alias()
+      .v6();
+    this.canAccessToElasticsearch = false;
```

Comment:
move these two lines to $onInit method. Its good practice to move all initialization to $onInit

Reply:
Removed, by full rework on this.